default.vcl

```diff
@@ -36,7 +36,7 @@ sub vcl_recv {
         set req.backend_hint = dynBackend.backend("public-things-api");
     }
     elif (req.url ~ "^\/content-preview\/.*$") {
-        set req.backend_hint = dynBackend.backend("content-public-read-preview");
+        set req.backend_hint = dynBackend.backend("unrolled-content-public-read-preview");
```

Comment:
It should probably be `unrolled-content-public-read`, as we decided to rename that service.